lib/ts/ink_align.h

```diff
@@ -53,6 +53,12 @@ union Alias64 {
 /** Default alignment */
 #define INK_ALIGN_DEFAULT(size) INK_ALIGN(size, INK_MIN_ALIGN)
 
+static inline size_t
+aligned_spacing(size_t len, size_t block = INK_MIN_ALIGN)
+{
+  return INK_ALIGN(len, block);
+}
```

Comment:
It's a delicate question as to "what is more complex to use".   Minimalism is just a heuristic that *may* lead to a simpler interface or may not.  In this case .. I find minimalism (deferring to legacy #defines) to be a problem.

Simplicity [which is not always minimal] and consistency is the aesthetic I'm looking for:  the name of a function relates clearly to what it does.

As for the verb:  alignment is for pointers ... by precise distance from address 0x00000000....

Alignment can be extended to elements from an aligned pointer, by spacing them larger than they require so *their* pointer matches an alignment similar to the first.   That's not alignment of a size ... but of the second pointer.